ui/components/app/wallet-overview/coin-buttons.tsx

```diff
@@ -363,48 +359,48 @@ const CoinButtons = ({
   return (
     <Box
       display={Display.Flex}
-      justifyContent={JustifyContent.spaceEvenly}
+      justifyContent={JustifyContent.spaceBetween}
       width={BlockSize.Full}
+      gap={4}
     >
       {
         ///: BEGIN:ONLY_INCLUDE_IF(build-main,build-beta,build-flask)
         <IconButton
           className={`${classPrefix}-overview__button`}
-          iconButtonClassName={iconButtonClassName}
           Icon={
             <Icon
               name={IconName.PlusAndMinus}
               color={IconColor.iconDefault}
-              size={IconSize.Sm}
+              size={IconSize.Md}
```

Comment:
Updating icon size to match Figma design

<img width="1098" alt="Screenshot 2025-05-16 at 11 08 18 AM" src="https://github.com/user-attachments/assets/e1eefccf-7c5f-4a0a-9aa4-7ca20d5d7cda" />